Client.php

```diff
@@ -129,13 +128,6 @@ protected function _run(HoaSocket\Node $node)
 
                     break;
 
-                case 433: // ERR_NICKNAMEINUSE
-                    throw new Exception(
```

Comment:
It was caught because `Exception` was extending `Hoa\Exception\Exception` that is extending `Hoa\Exception\Idle`. So the behavior is the same normally. And it is changed, it was a bug, so now it's good ;-).